sktime/dists_kernels/compose_tab_to_panel.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""
+Composers that create panel pairwise transformers from table pairwise transformers
+"""
+
+__author__ = ["fkiraly"]
+
+import numpy as np
+
+from sktime.dists_kernels._base import BasePairwiseTransformerPanel
+
+
+class AggrDist(BasePairwiseTransformerPanel):
+    """
+    pairwise distance based on aggregation function applied to sample-wise distance
+        example: AggrDist(ScipyDist()) is mean Euclidean distance between series
+
+    Components
+    ----------
+    transformer: pairwise transformer of BaseTrafoPw scitype (tabular pairwise)
```

Comment:
Minor:
```suggestion
    transformer: pairwise transformer of BaseTransformerPairwise scitype (tabular pairwise)
```

Reply:
Minor:
maybe call this `transformer` something like `tabular_transformer` or `transformer_tabular`?